LICENSE

```diff
@@ -0,0 +1,22 @@
+
+The MIT License (MIT)
+
+Copyright (c) UTN-LIS
```

Comment:
I think is OK we can update the file headers with this information and also we can add to the package.json file the same.